lib/plugins/aws/package/compile/events/apiGateway/lib/cors.js

```diff
@@ -24,6 +24,11 @@ module.exports = {
         'Access-Control-Allow-Methods': `'${config.methods.join(',')}'`,
         'Access-Control-Allow-Credentials': `'${config.allowCredentials}'`,
       };
+      
+      // Enable CORS Max Age usage if set
+      if (typeof config.maxAge !== 'undefined') {
```

Comment:
You can use lodash here `!_.isNil(config.maxAge) ...` and below. That will check for null and undefined.

Maybe even better would be to check for `!_.isInteger(config.maxAge)` afterwards too and emit a warning in that case.